src/display/q1_report_generator.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import click
+
+from config import logger
+from formula1_race_analysis.custom_exceptions import Formula1RaceAnalysisError
+from formula1_race_analysis.q1_session_analyzer import build_q1_report
+
+from .display_race_report import SortStrategy, display_race_report, filter_report, sort_report
+
+
+@click.command()
+@click.option("--data_dir", type=click.Path(exists=True, dir_okay=True, path_type=Path), required=True)
+@click.option(
+    "--order",
+    type=click.Choice(["asc", "desc"], case_sensitive=False),
+    default="asc",
+    show_default=True,
+    help="['asc'] = To sort results in ascending order, ['desc'] = To sort results in descending order",
+)
+@click.option("--driver", default=str)
+@click.option("--ignore_errors", is_flag=True, default=False)
+def generate_report(data_dir: Path, order: str, driver: str, ignore_errors: bool | None) -> None:
+    try:
+        logger.debug(f"Starting report generation. Data directory: '{data_dir}'.")
+        database = build_q1_report(Path(data_dir), ignore_errors)
+    except Formula1RaceAnalysisError as error:
+        logger.error(f"Failed during report generation: {type(error).__name__} - {error}")
+        click.get_current_context().exit(1)
+
+    sorted_report = []
+    logger.info("Processing F1 qualifying results.")
+    if driver:
```

Comment:
![image](https://github.com/user-attachments/assets/5f23a397-b276-4833-ac82-9e84419b61b9)
Please normalize it, you can't expect me to write properly